packages/calling/src/Contacts/ContactsClient.test.ts

```diff
@@ -217,7 +217,7 @@ describe('ContactClient Tests', () => {
     });
 
     expect(contactsResponse).toEqual({
-      data: codeObj.expectedData,
+      data: expect.any(Object),
```

Comment:
Changed this as a hack for the UT failing here. I'd like to understand how we reached the sequence for the `decryptText` array.

Reply:
Are we going to fix this as part of this PR?

Reply:
@rarajes2 no, this will come in a new [jira](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-549191)